app/consumer-democracy/app.go

```diff
@@ -475,21 +532,37 @@ func New(
 
 	skipGenesisInvariants := cast.ToBool(appOpts.Get(crisis.FlagSkipGenesisInvariants))
 
+	keeperMap := map[string]interface{}{
+		"/cosmos.gov.v1.MsgUpdateParams":               app.GovKeeper,
+		"/cosmos.bank.v1beta1.MsgUpdateParams":         app.BankKeeper,
+		"/cosmos.staking.v1beta1.MsgUpdateParams":      &app.StakingKeeper,
+		"/cosmos.distribution.v1beta1.MsgUpdateParams": app.DistrKeeper,
+		"/cosmos.mint.v1beta1.MsgUpdateParams":         app.MintKeeper,
+	}
+
+	app.KeeperMap = keeperMap
+
 	// NOTE: Any module instantiated in the module manager that is later modified
 	// must be passed by reference here.
 	app.MM = module.NewManager(
```

Comment:
I think the call is actually `module.NewBasicManager()` according to the SDK. @julienrbrt can you give more context?

Reply:
This looks good as it is. The basic manager is used for genesis stuff and is correctly defined above.